tests/apps/service_invocation/app.go

```diff
@@ -785,6 +786,62 @@ func grpcToGrpcTest(w http.ResponseWriter, r *http.Request) {
 	logAndSetResponse(w, http.StatusOK, "success")
 }
 
+// Performs calls from grpc client to grpc server.  It sends a random string to the other app
+// and expects the response to contain the same string inside an appResponse.
+func grpcWithoutHttpVerbToGrpcTest(w http.ResponseWriter, r *http.Request) {
```

Comment:
This function looks almost same as 'grpcToGrpcTest'. Suggest to reuse instead of dup.